slick-codegen/src/main/scala/slick/codegen/AbstractSourceCodeGenerator.scala

```diff
@@ -61,10 +61,18 @@ abstract class AbstractSourceCodeGenerator(model: m.Model)
       if(hlistEnabled) values.mkString(" :: ") + " :: HNil"
       else if (values.size == 1) values.head
       else if(values.size <= 22) s"""(${values.mkString(", ")})"""
-      else throw new Exception("Cannot generate tuple for > 22 columns, please set hlistEnable=true or override compound.")
+      else values.mkString("(" , " :: ", " :: HNil)")
+      //throw new Exception("Cannot generate tuple for > 22 columns, please set hlistEnable=true or override compound.")
```

Comment:
let's not leave stuff like this commented. Also the else branch can be merged with the first if, same right hand side